db_release_scripts/mosip_pms/sql/1.2.0.1-B3_to_1.2.0.1-B2_revoke.sql

```diff
@@ -0,0 +1,15 @@
+-- -------------------------------------------------------------------------------------------------
+-- Database Name: mosip_pms

```

Comment:
will this file name change as per the new upgrade convention followed?

Reply:
As per @ckm007 currently in develop branch changes will be with the old convention, when we bring the release-1.2.0.1 changes to develop, renaming of upgrade scripts will be taken up as a new task.